backend/src/main/kotlin/app/ehrenamtskarte/backend/auth/service/Authorizer.kt

```diff
@@ -25,4 +26,13 @@ object Authorizer {
             Role.REGION_ADMIN.db_value
         )
     }
+
+    fun mayViewUsersInProject(user: AdministratorEntity?, projectId: Int): Boolean {
+        return user?.projectId?.value == projectId && user.role == Role.PROJECT_ADMIN.db_value
```

Comment:
Why are we checking here `projectId.value` whereas below we are checking the `projectId` directly?

Reply:
Below, we are handling with `EntityID<Int>` on both sides; here we are handling with an `Int` and an `EntityID<Int>`.